.eslintrc.js

```diff
@@ -167,6 +167,15 @@ module.exports = {
     // TODO: Move back here the HTML formatting
     'vue/html-self-closing': 'off',
     'vue/no-v-html':         'error',
+
+    // FIXME: Disabled rules due upgrade to Vue 2.7.x and ESLint 9.x.x1
+    'vue/multi-word-component-names':             'off',
+    'vue/no-reserved-component-names':            'off',
+    'vue/no-useless-template-attributes':         'off',
+    'vue/attribute-hyphenation':                  'off',
+    'vue/valid-next-tick':                        'off',
+    'vue/no-computed-properties-in-data':         'off',
+    'vue/no-side-effects-in-computed-properties': 'off',
```

Comment:
I was able to run `yarn lint` with this commented out. Is this still needed?

Reply:
@mantis-toboggan-md guess what 😅

Reply:
I removed the `node_module` and reinstalled everything.
With the new rules no issues, while if I comment them out I get the error for all the rules I turned off:

```
✖ 422 problems (389 errors, 33 warnings)
  0 errors and 33 warnings potentially fixable with the `--fix` option.
```

Reply:
This was the issue when I tried it on push on my fork: https://github.com/cnotv/dashboard/actions/runs/4669320342/jobs/8267601781

Reply:
Alright, after deleting node_modules I see it too.